tests/test-theme-media.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package Create_Block_Theme
+ */
+class Test_Create_Block_Theme_Media extends WP_UnitTestCase {
+
+	public function test_make_images_block_local() {
+		$template          = new stdClass();
+		$template->content = '
+			<!-- wp:image -->
+			<figure class="wp-block-image"><img src="http://example.com/image.jpg" alt="Alternative Text" /></figure>
+			<!-- /wp:image -->
+		';
+		$new_template      = Theme_Media::make_template_images_local( $template );
+
+		// The image should be replaced with a relative URL
+		$this->assertStringNotContainsString( 'http://example.com/image.jpg', $new_template->content );
+		$this->assertStringContainsString( 'get_stylesheet_directory_uri', $new_template->content );
+		$this->assertStringContainsString( '/assets/images', $new_template->content );
+
+	}
+
+	public function test_make_cover_block_local() {
+		$template          = new stdClass();
+		$template->content = '
+			<!-- wp:cover {"url":"http://example.com/image.jpg"} -->
+				<div class="wp-block-cover">
+					<img class="wp-block-cover__image-background wp-image-628" alt="" src="http://example.com/image.jpg" data-object-fit="cover"/>
+					<div class="wp-block-cover__inner-container">
+					</div>
+				</div>
+			<!-- /wp:cover -->
+		';
+		$new_template      = Theme_Media::make_template_images_local( $template );
+
+		// The image should be replaced with a relative URL
+		$this->assertStringNotContainsString( 'http://example.com/image.jpg', $new_template->content );
```

Comment:
Maybe we could test the character encoding as part of this test.

Reply:
Regarding: 
> I noticed some characters in image alt test are not correctly encoded:

I tried a few tests and attempted to reproduce, but nothing I did seemed to change what was already in the template I was copying from.  Can you share what alt text was getting transferred incorrectly?

Can you share

Reply:
Yeah sure, it was this alt text: Test test "this is a" ><<""""'''T3sT

![image](https://github.com/WordPress/create-block-theme/assets/1645628/7218a21f-f462-4667-aaa2-1685ae335e15)

The less than/greater than characters are less of a priority, but I think the double quotes are probably worth handling.

Reply:
Ah wait, this is correct for HTML, right?! You'd want the HTML character codes to be used in this case. Sorry, ignore! I think the only issue here is lack of localization, but that can happen in a follow-up.

Reply:
The issue with php tags being turned into unicode is fixed by [f97a8a8](https://github.com/WordPress/create-block-theme/pull/522/commits/f97a8a81dd3d857af1305f8c9b4449e9c8eb0cb4) — The issue was that by moving assets to the theme as a first step when preparing the template for export, we were passing php code to WP functions that don't work with it, such as `serialize_blocks`.

Reply:
An encoding test has been added, and thank you hugely @vcanales for figuring out why it was busted!